### ISSUE TITLE
🇨🇳 Chinese TTS Arabic numerals and English letters cannot be recognized as speech

### ISSUE THREAD

**Issue by HarveyJiang** [TensorSpeech/TensorFlowTTS#604]
🇨🇳 Chinese TTS Arabic numerals and English letters cannot be recognized as speech， Only recognize Chinese characters。

**Comment by HarveyJiang** [comment]
use tts-tacotron2-baker-ch or tts-fastspeech2-baker-ch Only recognize Chinese characters。want Arabic numerals and English letters and Chinese characters, Do I need to retrain the model?

**Comment by dathudeptrai** [comment]
@HarveyJiang you need to retrain the model with your custom processor that supports both Chinese and English characters/symbols. And ofc, your data should contain both languages :D.

**Comment by HarveyJiang** [comment]
@dathudeptrai 
Can I understand this way, put Chinese training dataset A and English dataset B use tensorflow-tts-preprocess & tensorflow-tts-normalize to dump folder. then use "train_tacotron2.py" --train-dir /dataset/dump/train/ 【dump train contains Chinese and English train】.
Thanks
